test-result-summary-client/src/PerfCompare/lib/benchmarkVariantsInfo.js

```diff
@@ -7272,6 +7272,122 @@ export default
                 "higherbetter": "t",
                 "units": "req/sec"
             }
+        },
+        "17dev-4way-LargeThreadPool-Rampup": {
```

Comment:
Unlike Jenkins jobs, Perffarm jobs don't have a [BenchmarkMetricRouter](https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/master/TestResultSummaryService/parsers/BenchmarkMetricRouter.js 
) so we'll have some redundancy. Anyway, this is only temporary since we'll eventually stop using Perffarm.

Reply:
Actually let's use the same router for both Perffarm and non-Perffarm jobs as discussed today.